.genx/prompts/api.js

```diff
@@ -7,19 +7,12 @@ const ssoIntro = () => console.log(TEXTS.INTRO_API_SSO);
 module.exports = async (inquirer, prevAns = {}) => {
   apiHostIntro();
   const { apiHost = prevAns.apiHost } = await inquirer.prompt([
-    {
-      name: 'setApiHost',
-      type: 'confirm',
-      message: TEXTS.MESSAGE_API_SET_HOST,
-      when: !prevAns.apiHost,
-      default: true,
-    },
     {
       name: 'apiHost',
       type: 'input',
       message: TEXTS.MESSAGE_API_HOST,
-      when: ({ setApiHost }) => setApiHost,
-      default: prevAns.apiHost || 'ws://localhost/gwf/',
+      when: !prevAns.apiHost,
+      default: 'ws://localhost/gwf/',
```

Comment:
We can update the default to `ws://localhost:9064` while we're at it. That's what Create uses. It's more useful.

Reply:
will do

Reply:
addressed in https://github.com/genesiscommunitysuccess/blank-app-seed/pull/343/commits/d54a0373aa21f0883c7d80b7f74dc586ccbcec3b